README.md

```diff
@@ -1 +1,114 @@
 # Packer for Creating GraphDB GCP VM Image
+
+[Packer](https://www.packer.io/) configuration for building [GraphDB](https://www.ontotext.com/products/graphdb/) VM images
+for [Google Cloud](https://cloud.google.com/).
+
+## About GraphDB
+
+<p align="center">
+  <a href="https://www.ontotext.com/products/graphdb/">
+    <picture>
+      <img src="https://www.ontotext.com/wp-content/uploads/2022/09/Logo-GraphDB.svg" alt="GraphDB logo" title="GraphDB" height="75">
+    </picture>
+  </a>
+</p>
+
+Ontotext GraphDB is a highly efficient, scalable and robust graph database with RDF and SPARQL support. With excellent enterprise features,
+integration with external search applications, compatibility with industry standards, and both community and commercial support, GraphDB is
+the preferred database choice of both small independent developers and big enterprises.
+
+## What's Included
+
+The VM image includes the following libraries and software:
+
+- GraphDB
+- GraphDB Cluster Proxy
+- OpenTelemetry
+- Google Cloud Ops Agent
+- Google Cloud CLI
+
+## Usage
+
+**1. Authentication**
+
+Before being able to run Packer, you have to authenticate your local `gcloud` CLI in Google.
+You can follow the official documentation on this here https://cloud.google.com/docs/authentication/gcloud.
+
+**2. Variables**
+
+The Packer templates in this repository expect you to provide several required variables.
+Here's an example `variables.pkrvars.hcl` that you can extend and use:
+
+```hcl
+# Required
+project_id      = "your-google-cloud-project-id"
+build_zone      = "us-east1-b"
+graphdb_version = "10.7.0"
+
+# Optional
+graphdb_version         = "10.7.0"
+image_storage_locations = ["us"]
+```
+
+You check the rest of the variables in [variables.pkr.hcl](variables.pkr.hcl).
+
+**3. Running Packer**
+
+After preparing the variables, you can build VM images with:
+
+```shell
+packer build -var-file="variables.pkrvars.hcl" .
+```
+
+## Observability
+
+The image installs OpenTelemetry Java agent for auto instrumentation of GraphDB as well as Google's own Ops Agent for monitoring VMs.
+However, both of them are disabled by default, so to get them working you'd have to:
+
+**Enable Google Cloud Ops Agent**
+
+```shell
+systemctl disable google-cloud-ops-agent.service
```

Comment:
enable?

Reply:
we disable such tools by default, also in the aws / az packers because no point if you start a simple instance

Reply:
But these are the instructions on how to enable it and the next command is to start it.

Reply:
Ye, I copy/pasted it from the script and forgot to update it..